Framework/Core/src/FrameworkGUIDebugger.cxx

```diff
@@ -74,6 +76,24 @@ optionsTable(const DeviceSpec &spec, const DeviceControl &control) {
   ImGui::Columns(1);
 }
 
+ImVec4
+colorForLogLevel(LogParsingHelpers::LogLevel logLevel) {
+  switch (logLevel) {
+    case LogParsingHelpers::LogLevel::Info:
+      return PaletteHelpers::GREEN;
+    case LogParsingHelpers::LogLevel::Debug:
+      return ImVec4(153./255, 61./255, 61./255, 255./255);
+    case LogParsingHelpers::LogLevel::Warning:
+      return PaletteHelpers::DARK_YELLOW;
+    case LogParsingHelpers::LogLevel::Error:
+      return PaletteHelpers::DARK_RED;
```

Comment:
just noticed that this is difficult to read on the black background, would change to `RED`

Reply:
I will change this.